### PR TITLE
PAAS-3001: set tags before starting dd agent

### PR DESCRIPTION
**Pull request by julien-gagnon**
JIRA issue: https://jira.jahia.org/browse/PAAS-3001

Short description: Run tag script before starting dd agent on jcustomer

### CODE REVIEW

File: mixins/jcustomer.yml [hunk 1]
```diff
@@ -108,6 +108,7 @@ actions:
         if [ -f $file ]; then
           mv $file $file.disabled
         fi
+        /usr/local/bin/set_dd_tags.sh
         systemctl restart datadog-agent
```
Comment (Ronan-Gorain) [inline mixins/jcustomer.yml:112]: Maybe check the output of the script to know if restart needed or not. To avoid 2 restart in a row

Comment (julien-gagnon) [reply mixins/jcustomer.yml]: Thank you for the astute comment :) Added a check

Comment (laurentfufu) [reply mixins/jcustomer.yml]: I don't quite understand the point, isn't just running `set_dd_tags.sh` (and therefore removing the `systemctl restart`) enough?

Comment (julien-gagnon) [reply mixins/jcustomer.yml]: My understanding is that the cron might already have run so a restart might still  be necessary to take into account the modification to the config files.

Comment (Ronan-Gorain) [reply mixins/jcustomer.yml]: I understood like @julien-gagnon

Comment (laurentfufu) [reply mixins/jcustomer.yml]: indded, my bad :)

**Review by Ronan-Gorain — APPROVED** [review]

**Pull request merged**